Dockerfile

```
@@ -2,7 +2,7 @@ FROM mcr.microsoft.com/dotnet/runtime:7.0
 
```

```suggestion
FROM mcr.microsoft.com/dotnet/runtime:8.0
```

When running this locally, I seemed to need .NET runtime 8.0 as well since it fails to boot otherwise

---

Yup I have this working currently with this line being updated and the server_version as well.